content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,502 @@
+          set-privateKey-rotationPolicy.0-mutate-certificate-defaults.kyverno.io: added /spec/privateKey
+          set-revisionHistoryLimit.0-mutate-certificate-defaults.kyverno.io: added /spec/revisionHistoryLimit
+      creationTimestamp: "2023-08-29T17:36:35Z"
+      generation: 1
+      name: test-revision
+      namespace: default
+      uid: 11c2ccb0-e1a8-4b44-b09e-41c434c823fd
+    spec:
+      dnsNames:
+      - example.com
+      issuerRef:
+        group: cert-manager.io
+        kind: ClusterIssuer
+        name: not-my-corp-issuer
+      privateKey:
+        algorithm: ECDSA
+        encoding: PKCS1
+        rotationPolicy: Always
+        size: 521
+      revisionHistoryLimit: 2
+      secretName: test-revision-cert
+    ```
+
+1. We have successfully defaulted the `privateKey` and `revisionHistoryLimit` fields. Let's actually override all of these to validate that we can still set what we want as an end user. Here is a new `Certificate` resource to use fro testing called "test-revision-override":
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+      name: test-revision-override
+      namespace: default
+    spec:
+      dnsNames:
+      - example.com
+      issuerRef:
+        group: cert-manager.io
+        kind: ClusterIssuer
+        name: not-my-corp-issuer
+      privateKey:
+        algorithm: RSA
+        encoding: PKCS8
+        rotationPolicy: Never
+        size: 4096
+      revisionHistoryLimit: 44
+      secretName: test-revision-override-cert
+    ```
+
+    Dry-run apply this resource and validate it matches our `Certificate` YAML exactly:
+
+    ```shell
+    kubectl apply -f yamls/cert-test-revision-override.yaml --dry-run=server -o yaml
+    ```
+
+    An example output:
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/last-applied-configuration: |
+          {"apiVersion":"cert-manager.io/v1","kind":"Certificate","metadata":{"annotations":{},"name":"test-revision-override","namespace":"default"},"spec":{"dnsNames":["example.com"],"issuerRef":{"group":"cert-manager.io","kind":"ClusterIssuer","name":"not-my-corp-issuer"},"privateKey":{"algorithm":"RSA","encoding":"PKCS8","rotationPolicy":"Never","size":4096},"revisionHistoryLimit":44,"secretName":"test-revision-override-cert"}}
+      creationTimestamp: "2023-08-29T17:39:55Z"
+      generation: 1
+      name: test-revision-override
+      namespace: default
+      uid: 26637271-afb3-46b8-b4ce-eb3336ce1b9f
+    spec:
+      dnsNames:
+      - example.com
+      issuerRef:
+        group: cert-manager.io
+        kind: ClusterIssuer
+        name: not-my-corp-issuer
+      privateKey:
+        algorithm: RSA
+        encoding: PKCS8
+        rotationPolicy: Never
+        size: 4096
+      revisionHistoryLimit: 44
+      secretName: test-revision-override-cert
+    ```
+
```

Comment:
Stopping here. 

* All those commands above work well.
* I spotted another typo.
* Perhaps this first part of the content can be merged now and the content below can be added to a separate PR after 1.13 is released with your fix for the mutating webhook.
* Maybe write something about what "sane defaults" are recommended by the cert-manager team and explain why those are not the out-of-the-box defaults....here or in another PR?

Reply:
> Maybe write something about what "sane defaults" are recommended by the cert-manager team and explain why those are not the out-of-the-box defaults....here or in another PR?

I love that - will add a section. Might need your input after my first draft

> Perhaps this first part of the content can be merged now and the content below can be added to a separate PR after 1.13 is released with your fix for the mutating webhook.

I have another section to add with ingress-shim. So I'll see when I complete this, probably after 1.13 so I can clean up that setion, or make reference to.. run 1.13 or read here ..

Reply:
In the latest revision I am now able to move the <= v1.13 stuff to an appendix to make the tutorial flow better.
Added v1.14.X as a req for the tutorial.